.github/workflows/test.yml

```diff
@@ -34,7 +34,7 @@ jobs:
       - uses: actions/checkout@v4
       - uses: dtolnay/rust-toolchain@stable
       - name: Run internal tests
-        run: cargo test --verbose --features ${{ matrix.dialect }} ${{ matrix.signing }} -- --nocapture
+        run: cargo test --verbose --features arbitrary,${{ matrix.dialect }} ${{ matrix.signing }} -- --nocapture
```

Comment:
Should `arbitrary` be enabled together with all the dialect features?

Reply:
Not necessarily now, but it will be used in their tests, soon.